### ISSUE TITLE
Render fails on Debian 4.9.88

### ISSUE THREAD

**Issue by samtay** [faressoft/terminalizer#87]
There seems to be a number of issues coming from the underlying electron package. First time trying to `render` I found:
```
Error: 
  Error: [2773:0216/010033.435445:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /home/sam/.npm-global/lib/node_modules/terminalizer/node_modules/electron/dist/chrome-sandbox is owned by root and has mode 4755.
```
which luckily gave me a fix right in the error message. Now I'm getting
```
Error: 
  Error: [3037:0216/010239.315673:ERROR:buffer_manager.cc(488)] [.DisplayCompositor]GL ERROR :GL_INVALID_OPERATION : glBufferData: <- error from previous GL command
```
which appears to be an issue originating from electron, and can be solved by disabling gpu. I was able to fix this by cowboying in my `~/.npm-global/lib/node_modules/terminalizer/render/index.js` and adding the call:
```js
var path          = require('path'),
    app           = require('electron').app,
    BrowserWindow = require('electron').BrowserWindow,
    ipcMain       = require('electron').ipcMain,
    os            = require('os');

// Fix
app.disableHardwareAcceleration();
```
Obviously a hack, so I'm not gonna open a PR for that, but perhaps someone can dig into detecting when this is necessary and calling when needed.

Cheers.

**Comment by jdamata** [comment]
I had a similar problem running this on fedora 31:

First a fontconfig problem :
```
Error: 
  Error: Fontconfig warning: "/usr/share/fontconfig/conf.avail/05-reset-dirs-sample.conf", line 6: unknown element "reset-dirs"
```
Commenting out in  /usr/share/fontconfig/conf.avail/05-reset-dirs-sample.conf

Then the above fix by disabling hardware acceleration got me going.

**Comment by miraclx** [comment]
The underlying issue is described here
#102

And fixed on PR #97

**Comment by faressoft** [comment]
The issue appears to have been resolved and released in the new version v0.10.0 by #97. If you believe that there is still something missing or unresolved, please feel free to reopen it.
